code/datums/traits/neutral.dm

```diff
@@ -97,3 +97,21 @@
 
 /datum/quirk/monochromatic/remove()
 	quirk_holder.remove_client_colour(/datum/client_colour/monochrome)
+
+//MonkeStation Edit Start - Bald Quirk
+/datum/quirk/bald
+	name = "Bald"
+	desc = "Your hair seems to have gone missing. Luckily, you will spawn with a wig."
+	value = 0
+
+/datum/quirk/bald/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+	var/obj/item/clothing/head/wig/W = new /obj/item/clothing/head/wig
```

Comment:
For randomizing the hair
```suggestion
	var/obj/item/clothing/head/wig/random/W = new /obj/item/clothing/head/wig/random
```

Reply:
when calling new on a var with a defined type path, you dont need it twice

`var/obj/item/clothing/head/wig/random/W = new` or `new()` suffice